flagengine/organizations/models.go

```diff
@@ -0,0 +1,15 @@
+package organizations
```

Comment:
Can I be picky and ask that we stick to British english please? This should be `organisations`.

Reply:
:+1: